src/current/cockroachcloud/connect-to-a-serverless-cluster.md

```diff
@@ -104,7 +104,7 @@ For connection examples and code snippets in your language, see the following:
     `{database}`  | The name of the (existing) database.
 
   </section>
-  <section class="filter-content" markdown="1" data-scope="cockroachdb-client">
+  <section class="filter-content" markdown="1" data-scope="command-line">
```

Comment:
Looking at the CC UI and realizing the user sees "CockroachDB Client" here in the dropdown menu. I think we need to either specify that to use the command line instructions, they need to select this option from the menu or revert this back to CockroachDB Client